### PR TITLE
Setting up repo

### PR DESCRIPTION
**Pull request by randomeduc**
- Create React app with [CRA](https://create-react-app.dev/) using [cra-template-pwa](https://create-react-app.dev/docs/making-a-progressive-web-app/)
- Clean code from template.
- Setup husky, prettier and linter.

Related Issues
https://github.com/emeks-studio/elune-pwa/issues/1

### CODE REVIEW

File: src/reportWebVitals.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = (onPerfEntry) => {
```
Comment (mirko-plowtech) [inline src/reportWebVitals.js:1]: que era esto?

File: src/App.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```
Comment (mirko-plowtech) [inline src/App.test.js:5]: tmb out, no?

File: src/reportWebVitals.js (hunk 1 above)
Comment (randomeduc) [reply src/reportWebVitals.js]: Una cosa que trae React para medir performance de tu app, pareciera ser de google.

File: src/setupTests.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```
[the remark this answers is not in the document]
Comment (mirko-plowtech) [reply src/setupTests.js]: jajaja soy asi, bitziosa

**Review by mirko-plowtech — APPROVED** [review]
lgtm!

**Pull request merged**